### ISSUE TITLE
Namespaced map in deps.edn causes false positives

### ISSUE THREAD

**Issue by borkdude** [borkdude/clj-kondo#1093]
This should not give any warnings:

``` clojure
{:deps #:clj-kondo{clj-kondo {:mvn/version "2020.11.07"}}}
```
but it gives:
``` clojure
warning: Libs must be qualified, change {clj-kondo {:mvn/version "2020.11.07"}} => {clj-kondo {:mvn/version "2020.11.07"}}/{clj-kondo {:mvn/version "2020.11.07"}}
```

cc @seancorfield
